controller/appcontroller.go

```diff
@@ -2095,14 +2100,25 @@ func (ctrl *ApplicationController) newApplicationInformerAndLister() (cache.Shar
 				if err != nil {
 					return
 				}
+
 				var compareWith *CompareWith
+				var delay *time.Duration
+
 				oldApp, oldOK := old.(*appv1.Application)
 				newApp, newOK := new.(*appv1.Application)
-				if oldOK && newOK && automatedSyncEnabled(oldApp, newApp) {
-					log.WithField("application", newApp.QualifiedName()).Info("Enabled automated sync")
-					compareWith = CompareWithLatest.Pointer()
+				if oldOK && newOK {
+					if automatedSyncEnabled(oldApp, newApp) {
+						log.WithField("application", newApp.QualifiedName()).Info("Enabled automated sync")
+						compareWith = CompareWithLatest.Pointer()
+					}
+					if ctrl.statusRefreshJitter != 0 && oldApp.ResourceVersion == newApp.ResourceVersion {
+						// Handler is refreshing the apps, add a random jitter to spread the load and avoid spikes
+						jitter := time.Duration(float64(ctrl.statusRefreshJitter) * rand.Float64())
```

Comment:
I think we can use the k8s official jitter util for this instead of implementing custom one. https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait#Jitter wdyt?

Reply:
Calling the wait.Jitter() method would end up with this line instead 
`jitter := ctrl.statusRefreshJitter - wait.Jitter(ctrl.statusRefreshJitter, 1.0)`

IMO, I think the time it takes to understand why the jitter is subtracted and what is the meaning of 1.0 is greater than simply having the current code.

Reply:
Hmm makes sense